stan/math/rev/mat/functor/cvodes_ode_data.hpp

```diff
@@ -88,119 +124,41 @@ class cvodes_ode_data {
  private:
   inline void rhs(const double y[], double dy_dt[], double t) const {
```

Comment:
Please document this method. Especially the fact that the output of this function is `dy_dy`. Is there any reason the inputs are not first followed by the output?

Reply:
Doc is there. The order is aligned (along with other functions which were not quite straight).